src/server/cloak-server.spec.js

```diff
@@ -100,4 +101,17 @@ describe('cloak server', function() {
         cloakConfig.messages.createRoom('TEST_ROOM_NAME', user);
         expect(cloak.createRoom).toHaveBeenCalledWith('TEST_ROOM_NAME');
     });
+
+    it('joinRoom: finds the room that has the ID passed as argument', function() {
+        require('./cloak-server')({});
```

Comment:
This is called as the first thing in every test as far as I can tell, so it should probably be pulled out into either an existing beforeEach or a new one.